.github/workflows/tvos-compile-test.yml

```diff
@@ -0,0 +1,91 @@
+name: tvOS Compile EAS
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches: [main, 'sdk-*']
+    paths:
+      - .github/workflows/updates-e2e-basic.yml
+      - packages/@expo/cli/**
+      - packages/@expo/config-plugins/**
+      - packages/@expo/config-types/**
```

Comment:
Probably don't need a TypeScript type to trigger the full rebuild

Reply:
@EvanBacon yes we can pull that out, but we should probably add `metro-config`